packages/shared/src/runtime-utils/index.ts

```diff
@@ -17,11 +17,15 @@ export const DEFAULT_HIGHLIGHT_LANGUAGES = [
   ['mdx', 'tsx'],
 ];
 
+// TODO: these utils should be divided into node and runtime
```

Comment:
`isSCM` should be removed as it is an in-house feature

Reply:
I see. It's not in this pr~